src/main/scala/ru/tinkoff/gatling/profile/ProfileBuilderNew.scala

```diff
@@ -0,0 +1,82 @@
+package ru.tinkoff.gatling.profile
+
+import io.circe.yaml._
+import io.circe.generic.auto._
+import io.gatling.core.Predef._
+import io.gatling.core.structure.{ChainBuilder, ScenarioBuilder}
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import ru.tinkoff.gatling.utils.IntensityConverter.getIntensityFromString
+
+import scala.io.Source
+
+case class Params(method: Option[String], path: Option[String], headers: Option[List[String]], body: Option[String])
+
+case class Request(request: Option[String], intensity: Option[String], groups: Option[List[String]], params: Option[Params]) {
+
+  val requestIntensity: Double = getIntensityFromString(intensity.getOrElse("0 rps"))
+
+  def toRequest: HttpRequestBuilder = {
+    val uri: String                               = params
+      .getOrElse(throw new NoSuchElementException("No params in request"))
+      .path
+      .getOrElse(throw new NoSuchElementException("No path in params"))
```

Comment:
здесь наверное правильнее будет использовать пустой путь, чтобы обращения шли на baseUrl

```suggestion
      .getOrElse("")
```